packages/edit-site/src/components/page-pages/style.scss

```diff
@@ -42,4 +15,32 @@
 		// Windows High Contrast mode will show this outline, but not the box-shadow.
 		outline: 2px solid transparent;
 	}
+
+	&.edit-site-page-pages__media-wrapper {
+		width: $grid-unit-40;
```

Comment:
width and height were also updated to use `$grid-unit-40` instead of `$grid-unit-50`.